Terms of Reference.md

```diff
@@ -0,0 +1,140 @@
+# Terms of Reference
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Scope](#project-scope)
+- [Purpose of the Technical Committee](#purpose-of-the-technical-committee)
+- [Membership Composition](#membership-composition)
+- [Project Schedule](#project-schedule)
+- [Meeting Schedule](#meeting-schedule)
+- [Consensus Building](#consensus-building)
+- [Code of Conduct](#code-of-conduct)
```

Comment:
The table of contents needs to be updated. Some of the links are wrong and others are missing.

Reply:
Check.